src/components/icons/App.svelte

```diff
@@ -1,42 +1,26 @@
 <script>
-  let className = undefined;
-  export { className as class };
-  export let id = undefined;
-  export let tabindex = undefined;
-  export let focusable = false;
   export let title = undefined;
   export let style = undefined;
   export let dimension = 60;
 
-  $: ariaLabel = $$props["aria-label"];
-  $: ariaLabelledBy = $$props["aria-labelledby"];
-  $: labelled = ariaLabel || ariaLabelledBy || title;
+  let id = "icon-title-app";
   $: attributes = {
+    id,
     width: dimension,
     height: dimension,
-    role: labelled ? "img" : undefined,
-    focusable: tabindex === "0" ? true : focusable,
-    tabindex,
+    role: "img",
+    "aria-labelledby": id,
+    ...(style && { style }),
   };
 </script>
 
 <svg
-  on:click
-  on:mouseover
-  on:mouseenter
-  on:mouseleave
-  on:keyup
-  on:keydown
   xmlns="http://www.w3.org/2000/svg"
-  class="{className}"
   preserveAspectRatio="xMidYMid meet"
-  style="{style}"
-  id="{id}"
   viewBox="0 0 32 32"
-  aria-labelledby="iconTitle"
   {...attributes}
 >
-  <title id="iconTitle">All Apps</title>
+  <title id="{id}">{title ? title : "Icon"}</title>
```

Comment:
"Icon" isn't a meaningful title, especially if multiple SVG elements share the same title. Maybe make the default "App Icon", "Moon Icon", etc.

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/166/commits/9ff2407e77162347d964395260101aa905d46fd5